R/verb-count.R

```diff
@@ -26,6 +27,24 @@ count.tbl_lazy <- function(x, ..., wt = NULL, sort = FALSE, name = NULL) {
   out
 }
 
+#' @rdname count.tbl_lazy
+#' @importFrom dplyr add_count
+#' @importFrom lifecycle deprecated
+#' @export
+add_count.tbl_lazy <- function (x, ..., wt = NULL, sort = FALSE, name = NULL, .drop = deprecated()) {
```

Comment:
I've seen `lifecycle::deprecated()` in other places, not sure what the common practice is. Also, if it's unsupported, should this be simply `NULL` instead?

Reply:
I'm afraid I don't really know either.

Reply:
From `?deprecated`: "We recommend importing lifecycle::deprecated() in your namespace and use it without the namespace qualifier.

In general, we advise against such magical defaults, i.e. defaults that cannot be evaluated by the user. In the case of deprecated(), the trade-off is worth it because the meaning of this default is obvious and there is no reason for the user to call deprecated() themselves."

Reply:
I think this should simply be `NULL` as it is unsupported anyway.